mlflow/langchain/__init__.py

```diff
@@ -53,7 +55,8 @@
     _validate_and_prepare_target_save_path,
 )
 from mlflow.utils.requirements_utils import _get_pinned_requirement
-from mlflow.openai.utils import TEST_CONTENT
+
+from .retriever_chain import RetrieverChain
```

Comment:
```suggestion
```

We have `from mlflow.langchain.retriever_chain import RetrieverChain` on line 29.